modules/ROOT/pages/sap/sap-connector-xml-maven.adoc

```diff
@@ -0,0 +1,105 @@
+	xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:file="http://www.mulesoft.org/schema/mule/file" 
+	xmlns="http://www.mulesoft.org/schema/mule/core" 
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" 
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core 
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/file 
+	http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
+	http://www.mulesoft.org/schema/mule/http 
+	http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/ee/core 
+	http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+	http://www.mulesoft.org/schema/mule/sap 
+	http://www.mulesoft.org/schema/mule/sap/current/mule-sap.xsd">
+----
+
+== Add a POM File Dependency 
+
+The `pom.xml` Maven file generated by Anypoint Studio contains dependencies for a Mule app. When you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector. The Mule runtime engine translates the `LATEST` dependency keyword to the current version.
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule-sap-connector</artifactId>
+  <version>LATEST</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+To obtain the most up-to-date `pom.xml` file information, access the connector in 
+https://www.mulesoft.com/exchange/[Anypoint Exchange] and click *Dependency Snippets*.
+
+For JCo library support, add these dependencies:
+
+[source,xml,linenums]
+----
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.mule.tools.maven</groupId>
+            <artifactId>mule-maven-plugin</artifactId>
+            <version>${mule.maven.plugin.version}</version>
+            <extensions>true</extensions>
+            <configuration>
+                <sharedLibraries>
+                    <sharedLibrary>
+                        <groupId>com.sap.conn.jco</groupId>
+                        <artifactId>com.sap.conn.jco.sapjco3</artifactId>
+                    </sharedLibrary>
+                    <sharedLibrary>
+                        <groupId>com.sap.conn.idoc</groupId>
+                        <artifactId>com.sap.conn.idoc.sapidoc3</artifactId>
+                    </sharedLibrary>
+                    <sharedLibrary>
+                        <groupId>com.sap.conn.jco</groupId>
+                        <artifactId>com.sap.conn.jco.libsapjco3</artifactId>
+                    </sharedLibrary>
+                </sharedLibraries>
+            </configuration>
+        </plugin>
+    </plugins>
+</build>
+----
+
+== Next
+
+After completing your namespace and `pom.xml` file, you can try out the SAP xref:sap/sap-connector-examples.adoc[Examples].
```

Comment:
After manually coding your namespace and pom.xml file....

And what about the JCo library? Is that part of the POM file?

Reply:
try out > test, experiment with (internationalization)